backend/tracim_backend/lib/core/user.py

```diff
@@ -187,7 +194,7 @@ def get_known_user(
         )
         # INFO - G.M - 2018-07-27 - if user is set and is simple user, we
         # should show only user in same workspace as user
-        if self._user and self._user.profile.id <= Group.TIM_USER:
+        if filter_results and self._user and self._user.profile.id <= Group.TIM_USER:
```

Comment:
We should raise an exception if `filter_results and not self._user`

Reply:
Second point about `and self._user.profile.id <= Group.TIM_USER`: i'm not sure if `get_known_user` should continue to use this limitation because now, this is  ̀filter_results` which indicate to filter or not. If level of user must be check, it should be the responsibility of caller.

Reply:
> Second point about `and self._user.profile.id <= Group.TIM_USER`: i'm not sure if `get_known_user` should continue to use this limitation because now, this is ̀filter_results` which indicate to filter or not. If level of user must be check, it should be the responsibility of caller.

Doing this meaning add logic to controller, not sure that a good thing, we do have 4 case:
- filter_config = False and profile == 'user' -> no filter
-  filter_config = False and profile > 'user' -> no filter
-  filter_config = True and profile == 'user' -> apply filter
-  filter_config = True and profile > 'user' -> apply filter.